packages/webpack/src/plugin.js

```diff
@@ -452,7 +463,8 @@ class LavaMoatPlugin {
              * @returns {plugin is import('webpack').WebpackPluginInstance}
              */
             (plugin) =>
-              !!plugin && typeof plugin === 'object' && plugin.constructor.name === 'HtmlWebpackPlugin'
+              typeof plugin === 'object' &&
```

Comment:
former check covered a scenario where `plugin` is `null` thus avoiding a potential unhandled exception, isn't that important?

Reply:
It's a change suggested in review :see_no_evil: 
I accepted the suggestion and checked that typescript did not complain. but you're right, we should null-check. God knows what could be on the plugins list

Reply:
I knew it was there for that reason but saw typescript was fine with it and moved on. turns out it was just a lag to display a squiggle. It was complaining now.